plugin/index.js

```diff
@@ -85,6 +85,12 @@ module.exports = generators.Base.extend({
         name: 'npmVersion',
         message: 'NPM version used',
         default: version.major
+      }, {
+        type: 'confirm',
+        name: 'cssify',
+        message: 'Include CSSify?',
```

Comment:
I think we need either a short description on why they would want CSSify.  Maybe change this to "Include CSSify for Browserify support" or something more descriptive.